lib/scope.js

```diff
@@ -518,30 +506,30 @@ module.exports = {
     let page_data = { sought_var: '', fields: [] };
     const $ = cheerio.load( html );  // Could this be on `scope`?
 
-    // All these pieces of data feel very divorced from the field itself and the action happening.
-    // Surely there's a better way.
-
     // Get sought var
-    let encoded_sought_var = $($( `input[name="_event"]` )).attr( `value` );
-    // TODO: Change to `sought_name`?
-    let sought_var_str = await scope.fromBase64( scope, { base64_str: encoded_sought_var });
-    page_data.sought_var = JSON.parse( sought_var_str )[0];  // Why is this a list? Can multiple vars be sought?
+    let encoded_sought_var = $($( `#sought_variable` )).data( `variable` );  // This should come out to 'None' at the very least
```

Comment:
This is the new sought var element that developers will have to add to their interviews. AL will have them added automatically unless someone overrides the values there.

Reply:
This is how the developer can do it in their interview. If done this way (there are others, but this is one), this element should be on mobile as well. Not all methods do that.
https://github.com/plocket/docassemble-ALAutomatedTestingTests/blob/main/docassemble/ALAutomatedTestingTests/data/questions/all_tests.yml#L6-L10

Reply:
First comment added, along with link in warning message. Doesn't talk about mobile, but put that as something to add to docs in #259.